packages/styles/scss/components/ui-shell/header/_header.scss

```diff
@@ -42,6 +42,9 @@
     border: rem(1px) solid transparent;
     transition: background-color $duration-fast-02,
       border-color $duration-fast-02;
+    @include breakpoint-down('md') {
+      min-width: mini-units(6);
```

Comment:
Could we use the spacing tokens here? 

https://carbondesignsystem.com/guidelines/spacing/overview/#spacing-scale

Reply:
Let me try that

Reply:
Yup, done @tw15egan